### PR TITLE
docs: release v0.7.0 news

### PR DESCRIPTION
**Pull request by gnikit**
Adds v0.7.0 release into the webpage

## TODO

When this is done:
- [x] ~~Update the fpm.toml manifest to include the profiles~~ Deferred for when profiles are used in the backend not just manifest syntax

### CODE REVIEW

File: pages/news/2022/10-26-fpm-version-0.7.0.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+---
+author: Lewis McMillan, Damian Rouson, Giannis Nikiteas, Laurence Kedward, Sebastian Ehlert, Zuo Zhihua, Jakub Jelínek, Arteev Raina
+date: 2022-10-26
+category: release
+...
+
+# Fpm version 0.7.0 released
+
+This release introduces preprocessor support in the `fpm.toml` manifest, allowing
+for preprocessor macros, file suffixes and directories where the preprocessor should run.
```
Comment (milancurcic) [inline pages/news/2022/10-26-fpm-version-0.7.0.md:10]: Thank you for drafting this release note. I think it's in great shape, but only suggest acknowledging GSoC for the preproc table feature:

```suggestion
for preprocessor macros, file suffixes and directories where the preprocessor should run.
This feature was implemented as part of Arteev Raina's Google Summer of Code 2022 project.
```

**Review by milancurcic — APPROVED** [review]

**Comment by gnikit** [comment]
@milancurcic do we need to ping the i8 translation teams?

**Comment by awvwgk** [comment]
No need to ping anyone, just update the translation files and the translators will see them in weblate.

**Comment by gnikit** [comment]
> No need to ping anyone, just update the translation files and the translators will see them in weblate.

Would that be via `make gettext`? Also we need to cherry-pick the CI fix from [45e11a2](https://github.com/fortran-lang/fpm-docs/pull/87/commits/45e11a2069316c8a8439e3b54af376ab4b19e59d)

**Review by awvwgk — APPROVED** [review]
Thanks for writing it up, looks good to me.

**Pull request merged**